File: src/client/classes/api/req_res/update-nlog-response.ts
'use strict'

import { Nlog } from '@/classes/datas/nlog'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class UpdateNlogResponse extends GkillAPIResponse {

    updated_nlog: Nlog

    updated_kyou: Kyou | null

    constructor() {
        super()
        this.updated_nlog = new Nlog()
        this.updated_kyou = null
    }

}

File: src/server/gkill/api/req_res/update_nlog_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type UpdateNlogResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	UpdatedNlog *reps.Nlog `json:"updated_nlog"`

	UpdatedKyou *reps.Kyou `json:"updated_kyou"`
}
